expression/chunk_executor.go

```diff
@@ -64,6 +64,100 @@ func VectorizedExecute(ctx sessionctx.Context, exprs []Expression, iterator *chu
 	return nil
 }
 
+func evalOneVec(ctx sessionctx.Context, expr Expression, input *chunk.Chunk, output *chunk.Chunk, colIdx int) error {
```

Comment:
s/ evalOneVec  / evalOneColumnVec ?

Reply:
`evalOneVec` looks good to me. It's simple. `Column` is somehow equals to `Vector` semantically.

Reply:
ok.